app/views/layouts/application.html.erb

```diff
@@ -21,5 +21,6 @@
     </main>
   </div>
   <%= javascript_include_tag "application" %>
+  <%= javascript_include_tag "es6-components", type: "module" %>
```

Comment:
Don't suppose there's a way to write an integration test to confirm that "Grade X" browsers render content properly?

Reply:
Reading [our browser support docs](https://frontend.design-system.service.gov.uk/browser-support/#grade-x), it looks like we generally don't do any automated testing in Grade X browsers (or Grade C).

I've tested the page manually without JavaScript and it works fine, and I'm confident that the components are standalone and are built with progressive enhancement.

I'll rewrite the commit message to include a link to that ^ browser support page, to at least make it clearer what Grade X is